doc/specs/vulkan/chapters/pipelines.txt

```diff
@@ -1502,8 +1502,7 @@ bind point.
 The pipeline bound to ename:VK_PIPELINE_BIND_POINT_COMPUTE controls the
 behavior of flink:vkCmdDispatch and flink:vkCmdDispatchIndirect.
 The pipeline bound to ename:VK_PIPELINE_BIND_POINT_GRAPHICS controls the
-behavior of flink:vkCmdDraw, flink:vkCmdDrawIndexed,
-flink:vkCmdDrawIndirect, and flink:vkCmdDrawIndexedIndirect.
+behavior of all ftext:vkCmdDraw* commands.
```

Comment:
Minor: Might be better to do <<draw, draw commands>>? We use that elsewhere.

Reply:
Agree with Tobias' suggestion, otherwise this PR looks good to me.

Reply:
So be it!